.github/workflows/release-apt-get.yml

```diff
@@ -0,0 +1,52 @@
+name: "release-apt-get"
+on:
+  release:
+    types: [released]
+
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release Tag'
+        required: true
+        default: 'latest'
```

Comment:
Just so I understand, this workflow triggers when we publish a release that becomes the "latest" release. We don't really use any information about that release from the workflow startup details because `fetch_release.py` will download from the public feed and push whatever is found there.

I'm not sure if this indirection is _necessary_ or if there is a "blessed" way to do it in GitHub Actions. I don't care that much as long as it works (and I understand your approach here).

Reply:
This is a special (and very handy!) action that allows us to run the workflow manually. I can't give an example from microsoft/git yet, as the code has not made it into the default branch. However, here's what this looks like in GCM Core:

![image](https://user-images.githubusercontent.com/11321782/122833339-a6816e80-d2a1-11eb-86c5-dd34aa784361.png)

This will not impact the workflows that trigger automatically when a new release becomes available.

Reply:
Here is some further reading if you'd like to learn more!
https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow